src/xcode/ENA/ENA/Source/Client/HTTP Client/URLSession+Convenience.swift

```diff
@@ -38,18 +38,26 @@ extension URLSession {
 		request.httpMethod = "POST"
 		request.httpBody = body
 		request.setValue("application/json", forHTTPHeaderField: "Content-Type")
-		response(for: request, completion: completion)
+		response(for: request, isFake: false, completion: completion)
 	}
 
 	// This method executes HTTP requests.
 	// It does some additional checks - purely for convenience:
 	// - if there is an error it aborts
 	// - if there is either no HTTP body and/or HTTPURLResponse it aborts
+	// Note that, if we send out a fake request, we omit
+	// the response and give back a fakeResponse failure.
 	func response(
 		for request: URLRequest,
+		isFake: Bool = false,
```

Comment:
Do we need to have this `isFake` parameter here? If we only need this for testing could we use inheritance or other methods to test this method?